src/_zkapauthorizer/schema.py

```diff
@@ -208,4 +208,18 @@ def run_schema_upgrades(upgrades, cursor):
         ALTER TABLE [unblinded-tokens] ADD COLUMN [redemption-group] integer DEFAULT 1
         """,
     ],
+    6: [
+        """
+        -- track the "event-stream" which are a series of SQL statements
+        -- that modify the database (except statements which modify this table?)
```

Comment:
the exception here is correct.  the capture layer will have to know about this table and exclude it from capturing, otherwise I assume you have the very predictable infinite recursion.